src/alire/alire-containers.ads

```diff
@@ -31,12 +31,12 @@ package Alire.Containers with Preelaborate is
                                              Releases."=");
    subtype Release_H is Release_Holders.Holder;
 
-   package Project_Release_Maps is new Ada.Containers.Indefinite_Ordered_Maps
-     (Alire.Project, Releases.Release, "<", Releases."=");
-   type Release_Map is new Project_Release_Maps.Map with null record;
+   package Name_Release_Maps is new Ada.Containers.Indefinite_Ordered_Maps
```

Comment:
Shouldn't this be `Crate_Release_Maps`?

Reply:
Sometimes I have difficulties choosing between name and crate depending on context. I change it if you find it more consistent.

Reply:
From what I know of the code it makes more sense to me.

Reply:
Done in e07df2a